apps/regression/model/linear_model.h

```diff
@@ -1,5 +1,5 @@
-#ifndef REGRESSION_LINEAR_MODEL_H
-#define REGRESSION_LINEAR_MODEL_H
+#ifndef REGRESSION_AFFINE_MODEL_H
```

Comment:
The header guardians should match the file name, so REGRESSION_LINEAR_MODEL_H here